### PR TITLE
refactor: add 'musl' as runtime dependency for 'base'

### PR DESCRIPTION
**Pull request by smira**
This dries up dependency tree in `pkg.yaml` and graphs.

See https://github.com/talos-systems/bldr/pull/33

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Comment by smira** [comment]
WIP until bldr PR is merged and shebang can be updated

File: base/pkg.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ shell: /toolchain/bin/bash
 dependencies:
   - image: docker.io/autonomy/tools:7de4db2
   - stage: musl
+    runtime: yes
```
Comment (andrewrynhard) [inline base/pkg.yaml]: Cool, so this says "anytime `base` is a dependency, pull `musl` in too"?

Comment (smira) [reply base/pkg.yaml]: yep, exactly, and it works transitively and deduplicates stages as required.

we can actually pull this `tools` dependency as runtime and avoid putting it in every stage

I'm torn whether it's nice or not

Comment (smira) [reply base/pkg.yaml]: actually it's already pulled similar way by `finalize` which does copy `/` -> `/`, so `base` contains `tools`

**Pull request merged**